ot/unbalanced.py

```diff
@@ -376,6 +394,16 @@ def sinkhorn_knopp_unbalanced(a, b, M, reg, reg_m, numItermax=1000,
     else:
         n_hists = 0
 
+    if isinstance(reg_m, float) or isinstance(reg_m, int):
```

Comment:
or len(reg_m) == 1 ?

if it us a numpy or torch array of length 1 it shoud also work, please check